### PR TITLE
Expose imported attributes in metrics.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

This PR depends on vespa-engine/vespa#31061

### CODE REVIEW

File: tests/search/metrics/child.sd [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+schema child {
+  document child {
+    field f1 type string {
```
Comment (geirst) [inline tests/search/metrics/child.sd:4]: Is this field really needed for the test? If not, please remove.

File: tests/search/metrics/child.sd [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      indexing: index | summary
+    }
+    field f2 type int {
```
Comment (geirst) [inline tests/search/metrics/child.sd:7]: Is this field really needed for the test? If not, please remove.

File: tests/search/metrics/child.sd [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      indexing: attribute
+    }
+    field ref type reference<parent> {
+      indexing: attribute | summary
+    }
+  }
+
+  import field ref.f3 as my_f3 {}
+
+  rank-profile default {
```
Comment (geirst) [inline tests/search/metrics/child.sd:17]: Is the rank profile really needed for the test? If not, please remove.

File: tests/search/metrics/metrics.rb [hunk 4]
```diff
@@ -83,6 +83,23 @@ def test_metrics
     assert_document_db_attribute_memory_usage(metrics)
   end
 
+  def test_metrics_imported_attributes
+    set_description("Check reporting of metrics with imported attributes")
+    deploy_app(SearchApp.new.cluster(SearchCluster.new('test').
+                      sd(selfdir + "parent.sd", { :global => true }).
+                      sd(selfdir + "child.sd").
+                      visibility_delay(0.001).
+                      tune_searchnode({:summary => {:store => {:cache => { :maxsize => 8192,
```
Comment (geirst) [inline tests/search/metrics/metrics.rb:92]: Is this tuning really needed for the test? If not, please remove. Applies for visibility_delay() as well.

**Review by geirst — APPROVED** [review]
👍

**Review by baldersheim — APPROVED** [review]

**Pull request merged**